runtime/rococo/src/weights/mod.rs

```diff
@@ -15,9 +15,23 @@
 
 //! A list of the different weight modules for our runtime.
 
-pub mod runtime_parachains_configuration;
+pub mod frame_system;
```

Comment:
This is 100% the wrong fix, `frame_system` is a Rust crate that is imported and can be seen in `Cargo.toml`.

Reply:
Removed. Worth noting that this is in Westend as well here:
- https://github.com/paritytech/polkadot/blob/b567c8bfa760a1fc57e980494de18bf7182ae3dd/runtime/westend/src/weights/mod.rs#L18

Reply:
I am using Westend as a base to mirror Rococo.

Reply:
This is fine since it is a `pub mod`, not `pub use` so it will use the local `runtime/rococo/src/weights/frame_system.rs` for that once it exists.

Reply:
Oh wait, I mistook this file's purpose as trying to redefine the `frame_system` pallet, while in fact it's simply trying to provide the weights for `frame_system`. No need to remove it then.